src/Responses/PutRecordingTextTrackResponse.php

```diff
@@ -24,6 +24,11 @@
  */
 class PutRecordingTextTrackResponse extends BaseResponseAsJson
 {
+    const KEY_SUCCESS     = 'upload_text_track_success';
+    const KEY_FAILED      = 'upload_text_track_failed';
+    const KEY_EMPTY       = 'empty_uploaded_text_track';
+    const KEY_PARAM_ERROR = 'paramError';
```

Comment:
I think a global list of messageKeys in BBB would have prevented this naming scheme chaos